flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/service/operation/OperationExecutor.java

```diff
@@ -193,6 +194,7 @@ public ResultFetcher configureSession(OperationHandle handle, String statement)
                     String.format(
                             "Unsupported statement for configuring session:%s\n"
                                     + "The configureSession API only supports to execute statement of type "
+                                    + "SET, RESET, "
```

Comment:
nit: the previous line's english is not great. I suggest:
`The configureSession API only supports executing statements of type `

Reply:
nit: could we put all of these values into a variable we just iterate. Ideally the variable would be derived from what the API uses to validate.